CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+This page contains instructions on how to propose and implement feature changes to Presto.
+
+# The Request for Comments
+
+## Overview of the process
+
+```mermaid
+graph TD;
+    id1["Create an RFC"]-->id2["Get feedback on the RFC"];
+    id2["Get feedback on the RFC"]-->id3["RFC is closed"];
+    id2["Get feedback on the RFC"]-->id4["RFC is merged"];
+    id4["RFC is merged"]-->id5["Create a Github issue linking the RFC"];
+    id5["Create a Github issue linking the RFC"]-->id6["Create PRs implementing the RFC (link to the issue)"];
+```
+
+## Step 1: Create an RFC
+RFCs are located in their own repository.
+
+To create one:
+
+1. Fork the https://github.com/prestodb/rfcs repository
+2. Copy the template file `RFC-0000-template.md` to `RFC-00xx-your-feature.md` and fill it out with your proposal.
+   The template is a guideline, feel free to add sections as appropriate
+3. You may also have the template simply link to another editor, like a Google Docs file, but please ensure that the
+   document is publicly visible.  This can make the template easier to add edit, but commenting doesn’t scale very well, so
+   please use this option with caution.
+
+## Step 2: Get Feedback on the RFC
+1. Submit a pull request [to the main repository](https://github.com/prestodb/rfcs) from your fork titled 
+`RFC-00xx-your-feature.md`
+2. Before your PR is ready for review,
+   [mark the PR as draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft).
+3. Once it’s ready for review, remove the draft label.
```

Comment:
We may want to rephrase this for clarity since it seems like we won't be using traditional GH labels for our process

```suggestion
3. Once it’s ready for review, [move the PR out of draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review).
```

Reply:
Thanks!